docs/faq.md

```diff
@@ -0,0 +1,10 @@
+# General FAQ
+
+- AMD GPUs: Changing resolution, using DRI_PRIME and doing various other things can cause crashes, but `echo high| sudo tee /sys/bus/pci/drivers/amdgpu/0000:??:??.?/power_dpm_force_performance_level` or adding the `amdgpu.dpm=0` to the kernel commandline stops these crashes.
+- MacBook fans running up, screen freezing, then it turns off: T2 panic; try a patched kernel.
+- WiFi GUI not working: try kernel `5.15.12`.
+- When should I use the Big Sur kernel and when should I use the Mojave kernel? idk answer this for me
```

Comment:
As we can use Asahilinux's wifi patchset, we shouldn't need two kernels anymore, but we do need to get kernels updated to use that patchset.